netutils/ping.py

```diff
@@ -1,8 +1,9 @@
 """Functions to create a ping via pure Python."""
 import socket
+import typing as t
 
 
-def tcp_ping(ip, port, timeout=1):  # pylint: disable=invalid-name
+def tcp_ping(ip: str, port: int, timeout: t.Optional[int] = 1) -> bool:  # pylint: disable=invalid-name
```

Comment:
Can `timeout` be `None` in any case? If not, we can also get rid of the below ignore comments